AdobeStockAsset/Test/_files/objects/category.php

```diff
@@ -0,0 +1,23 @@
+<?php
```

Comment:
I think that having additional `objects` folder is excessive here, can we get rid of it and have fixtures simply like `_files/category.php`, `_files/keywords.php` etc?